tensorboard/plugins/interactive_inference/utils/inference_utils.py

```diff
@@ -125,7 +125,9 @@ class OriginalFeatureList(object):
   def __init__(self, feature_name, original_value, feature_type):
     """Inits OriginalFeatureList."""
     self.feature_name = feature_name
-    self.original_value = original_value
+    self.original_value = [
```

Comment:
I think it's better to use ensure_text from six for consistency with https://github.com/tensorflow/tensorboard/pull/2311.

Reply:
done

Reply:
actually can't as tensorboard/tensorflow depend on six 1.10.0 which doesn't include the ensure_* functions. our separate witwidget code seems to pull in six 1.12.0 due to transitive dependencies that is why #2311 works for us. so i reverted back to the first implementation and added an explicit dependency in witwidget setup.py on six 1.12.0 to improve the pip package setup dependencies.

Reply:
I see, sounds good then.